### PR TITLE
Implement explorer, a set of simple commands to view cluster information

### PR DESCRIPTION
**Pull request by AstroProfundis**
This adds several features under `show` sub-command to display basic information of a TiDB cluster.

All contents are from collected data by insight itself, only static & basic information are implemented.

Available commands are:

 - `show servers` / `show server <server alias>`: List / Show basic information of server
 - `show tidb/tikv/pd`: Show basic information of specific TiDB module
 - `show summary`: Sum of `show tidb/tikv/pd`, plus application version statics for each module

### CODE REVIEW

File: collector/collector.go [hunk 1]
```diff
@@ -43,9 +44,24 @@ type Metrics struct {
 	ProcStats  []ProcessStat   `json:"proc_stats"`
 }
 
+type options struct {
+	Pid string
+}
+
+func parseOpts() options {
+	optPid := flag.String("pid", "0", "The PID of process to collect info. Multiple PIDs can be seperatted by ','.")
```
Comment (ethercflow) [inline collector/collector.go]: Is it better to use "" instead of "0" ?

File: collector/process.go [hunk 2]
```diff
@@ -29,20 +30,39 @@ type RlimitUsage struct {
...
-		if proc == nil {
-			continue
+	} else {
+		tiServers := []string{"pd-server", "tikv-server", "tidb-server"}
+		for _, procName := range tiServers {
+			proc, err := getProcessesByName(procName)
+			if err != nil {
+				log.Fatal(err)
+			}
+			if proc == nil {
```
Comment (ethercflow) [inline collector/process.go]: It seems inconsistent with the above behavior? [#46](https://github.com/pingcap/tidb-insight/pull/44/files#diff-82a95ed34339398b0ebfe0dc33ef3116R46)

Comment (ethercflow) [reply collector/process.go]: BTW, if the loop's logics are same,  move `getProcessByPID`, `getProcessesByName` and itered array as func's params can remove dup.

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+ansible>=2.5
```
Comment (ethercflow) [inline requirements.txt]: Need ,<2.7.0 or not? @LinuxGit

File: collector/process.go (hunk 2 above)
Comment (AstroProfundis) [reply collector/process.go]: If no process matched, `proc` might be empty, but no error have occurred.

**Pull request merged**